features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/package.json

```diff
@@ -30,7 +30,7 @@
     "react-router-dom": "^4.1.1",
     "react-tap-event-plugin": "^2.0.1",
     "react-toastify": "^1.7.0",
-    "swagger-client": "3.0.19"
+    "swagger-client": "git://github.com/RenukaFernando/swagger-js.git"
```

Comment:
Do we need this?

Reply:
Yes @malinthaprasan , I have asked to do so in https://github.com/wso2/carbon-apimgt/pull/4809#discussion_r155800107 here, since another method of adding a patched version of node module is include it as an external module, which is somewhat messy, We can revert this  when @swagger-api people accept this change https://github.com/swagger-api/swagger-js/pull/1189 hopefully :blush:

Reply:
Okay cool :blush:

Reply:
Bdw looks like https://github.com/swagger-api/swagger-js/pull/1189 has conflicts with their master branch

Reply:
Conflict resolved in the PR: https://github.com/swagger-api/swagger-js/pull/1189; Thank you...